### DISCUSSION TITLE
thanhle.blog/quick-note-tai-sao-khong-bo-nhieu-trung-vao-cung-mot-ro

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#82] [Ideas]
# thanhle.blog/quick-note-tai-sao-khong-bo-nhieu-trung-vao-cung-mot-ro

Câu nói mọi người hay nghi khi đầu tư là không được bỏ nhiều trứng vào cùng một rổ, hay là nên chia ra nhiều mà không all in vào một thằng?

https://thanhle.blog/blog/quick-note-tai-sao-khong-bo-nhieu-trung-vao-cung-mot-ro

**Comment by hai123vn** [comment]
Anh ơi khi nào anh ra thêm nhiều bài mới vậy ạ, ngày nào cũng dô hóng anh ra bài mới để đọc. Mấy bài anh viết đọc cuốn quá 😊😊😊😊

**Reply by thanhlmm** [reply]
em thích đọc về chủ đề gì nhỉ 🤔 để anh coi thử

**Reply by hai123vn** [reply]
Dạ cái gì cũng được anh ơi, tại em thấy anh làm về chủ đề gì đọc cũng cuốn và thú vị hết á 🤣
